tests/inference/test_modeling.py

```diff
@@ -13,1884 +13,1512 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import gc
-import os
```

Comment:
Why so many lines were removed and other commented out ?

Reply:
Good catch, I accidentally remove them for debugging a specific test.... Putting it back.